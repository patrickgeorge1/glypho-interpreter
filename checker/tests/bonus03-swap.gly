===yNNN9ttt4nnFFHHHu||**Z000vnnnBNNNg(((