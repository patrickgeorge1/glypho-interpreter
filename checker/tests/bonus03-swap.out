1B2B13345265AAC8443J245325CC43435E246453
-3AJ458
0
5BBC98AA742312323EF236D69100J
